sql/filings-by-zip-since-0323.sql

```diff
@@ -40,11 +39,13 @@ select a.zipcode,
 filings_since_032320, 
 -- total residential units in the zip code as per PLUTO
 unitsres_total, 
--- total residential units in the zip code except for single unit properties
-unitsres_2,
--- filings normalized by total res units in the zip code
-filings_since_032320 * 1000 / nullif(unitsres_total, 0)::numeric as filingsrate_total, 
--- filings normalized by total res units in the zip code except for single unit properties. 
-filings_since_032320 * 1000 / nullif(unitsres_2, 0)::numeric as filingsrate_2plus
+--total residential units in the zip code as per PLUTO
+unitsrental,
+--total residential units in the zip code except for single unit properties
+filings_since_032320 * 1000 / nullif(unitsrental, 0)::numeric as filingsrate_2plus
+--filings normalized by total res units in the zip code except for single unit properties. 
```

Comment:
Oh it's these column renames that are causing `yarn build-data` to fail.  If you either change the names back, or modify `lib/filings-by-zip/data.ts` to use the new names, it will work again.

Also though, I'm noticing that `unitsrental` has the same comment above it as `unitsres_total`, is that accurate?  (In your original PR your descriptions of the columns were put below the SQL for the column but I moved them above in a later commit BTW, so sorry if that part is confusing.)

Reply:
yay! thank you for figuring this out! I edited data.ts and fixed the comments and pushed to development